target/linux/bcm53xx/patches-5.10/341-ARM-dts-NSP-pinctrl-avoid-unnecessary-probe-deferrals.patch

```diff
@@ -0,0 +1,46 @@
+From: Chris Packham <chris.packham@alliedtelesis.co.nz>
+Date: Fri, 25 Oct 2019 17:00:41 +1300
+Subject: [PATCH 2/2] ARM: dts: NSP: avoid unnecessary probe deferrals
+
+The pinctrl node is used by the gpioa node. Which may have more
+descendants at a board level. If the pinctrl node isn't probed first the
+gpio is deferred and anything that needs a gpio pin on that chip is also
+deferred.
```

Comment:
You described what happens but you didn't explain actualy problem. Can you please include info what deferring is actually an issue? What actual problem this hack fixes?

Reply:
This patch has been removed as the underlying issue no longer occurs.